content/get-involved.ts

```diff
@@ -77,27 +82,42 @@ const getInvolvedContent: GetInvolvedContent = {
     imgsrc: 'illustrations/climbing-guy.svg',
     mobileImgsrc: 'illustrations/checklist1.svg',
   },
-  volunteerPage: {
-    header: 'Help us break down barriers by joining the team.',
-    body: [
-      'You\'ll be a perfect fit if you:',
-    ],
-    facts: [
+  perfectFit: {
+    header: 'You\'ll be a perfect if...',
```

Comment:
So the figma screenshot shows 2 different headers: "You'll be a perfect fit if" and "You'll be perfect if". I think we have a mixture of the 2 here 😂

Reply:
LOL sorry! the TG had decided to implement the top header, so it has been refactored to `You'll be a perfect fit if`